modules/chapter2/pages/section4.adoc

```diff
@@ -0,0 +1,141 @@
+
+To perform this exercise, ensure that your platform meets the following conditions:
+
+* An S3 bucket.
+* The `data-scientists` user group exists in OpenShift.
+* The `user4` and `user5` users exist in OpenShift and are assigned to the `data-scientists` group.
+
+The identity provided of your demo platform already includes these users.
+To create the group, log in as an administrator user in OpenShift and then apply the https://github.com/RedHatQuickCourses/rhods-qc-apps/blob/ch02s04-lab/1.intro/chapter2/lab/group.yaml file.
+
+If you use the `oc` CLI in your computer, then you can run:
+
+[subs=+quotes]
+----
+$ *oc create -f \
+https://github.com/RedHatQuickCourses/rhods-qc-apps/blob/ch02s04-lab/1.intro/chapter2/lab/group.yaml*
+----
+
+// TODO adjust yaml URL after merging to main
+====
+
+== Specifications
+
+* Create two data science projects as `user1`: `intro-projects-a` and `intro-projects-b`.
+
+* Create a workbench in `intro-projects-a`.
+  - Use the `Standard Data Science` image.
+  - This workbench must be connected to the S3 bucket that is available in your demo platform.
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_connection.ipynb` notebook in this workbench.
+
+* Create a workbench in `intro-projects-b`.
+  - Select any image for this workbench.
+  - Pass your name in the `STUDENT_NAME` environment variable.
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_env_vars.ipynb` notebook in this workbench.
+
+* The `data-scientists` group must have edit permissions in `intro-projects-a`.
+
+* Only the `user4` user must have edit permissions in `intro-projects-b`.
+
+
+== Solution
+
+1. Create the projects.
+
+a. From your RHODS dashboard, click btn:[Data Science Projects] in the left navigation pane.
+
+b. In the btn:[Data science projects] page, click btn:[Create data science project].
+
+c. Enter `intro-projects-a` as the project name and click btn:[Create].
+
+d. Repeat the same steps to create `intro-projects-b`.
+
+
+2. Create a workbench connected to S3 in `intro-projects-a`.
+
+a. Navigate the `intro-projects-a` dashboard and click btn:[Create workbench].
+In the workbench creation page, enter the workbench name, select the workbench image, and the image version.
+
+b. Select btn:[Tiny] as the container size and 0 GPUs.
+
+c. In the btn:[Data connections] section, select btn:[Use a data connection].
+
+d. Enter the configuration values of your S3 connection.
+
+e. Click btn:[Create workbench] to finish the creation of the workbench and the data connection.
+
+
+3. Verify the connection of the workbench in `intro-projects-a`.
+
+a. Open the workbench of the `intro-projects-a` project.
+
+b. Log in to JupyterLab and accept the access form.
+
+c. Clone the https://github.com/RedHatQuickCourses/rhods-intro repository.
```

Comment:
Move to qc-apps later?

Reply:
This is already in https://github.com/RedHatQuickCourses/rhods-qc-apps/pull/2 

I will adjust the repo URL.